lib/haxfred-slack.js

```diff
@@ -1,4 +1,9 @@
 var Slack = require('slack-client');
+
+var on_open = require('./on_open');
```

Comment:
I camel case for non constants personally, but I don't feel super strongly enough about it here to put up a fight :) As long as we are consistent one way or another across projects we are good :)